libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/WalkingOptionsNavigation.kt

```diff
@@ -0,0 +1,95 @@
+package com.mapbox.navigation.base.route.model
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Class for specifying options for use with the walking profile.
+ *
+ * @property walkingSpeed - Walking speed in meters per second. Must be between 0.14 and 6.94 meters per second.
+ * Defaults to 1.42 meters per second
+ *
+ * @property walkwayBias - A bias which determines whether the route should prefer or avoid the use of roads or paths
+ * that are set aside for pedestrian-only use (walkways). The allowed range of values is from
+ * -1 to 1, where -1 indicates indicates preference to avoid walkways, 1 indicates preference
+ * to favor walkways, and 0 indicates no preference (the default).
+ *
+ * @property alleyBias - A bias which determines whether the route should prefer or avoid the use of alleys. The
+ * allowed range of values is from -1 to 1, where -1 indicates indicates preference to avoid
+ * alleys, 1 indicates preference to favor alleys, and 0 indicates no preference (the default).
+ */
+data class WalkingOptionsNavigation(
```

Comment:
Does this need to be a `data class` if we're already implementing a builder?

Reply:
data class is just sugar code, we can keep it